README.md

```diff
@@ -1,74 +1,22 @@
 <p align="center">
     <a href="https://laravel.com" target="_blank"><img src="https://www.supportpal.com/assets/img/logo_blue_small.png" /></a>
-    <br>
+    <br />
```

Comment:
Why's this necessary? `<br>` is HTML5, `<br />` is XHTML

Reply:
Didn't realise this had actually become valid, changed it back.